src/index.prebid.js

```diff
@@ -0,0 +1,141 @@
+import OpenAds from '@schibstedspain/openads'
+import AppNexusConnector from '@schibstedspain/openads-appnexus-prebid'
```

Comment:
We use AppNexusConnector in AppNexus connector implementation, maybe it would be better to use AppNexusConnectorPrebid to avoid confusion because we have both connectors in the same repository.

Reply:
Changed ;)